libsolidity/ast/ASTAnnotations.h

```diff
@@ -66,6 +66,7 @@ struct DocTag
 {
 	std::string content;	///< The text content of the tag.
 	std::string paramName;	///< Only used for @param, stores the parameter name.
+	bool inherited = false; ///< True if this tag was inherited
```

Comment:
How is this used?

Reply:
Ah, I think that was from a different attempt and was kept. Will remove.